### ISSUE TITLE
Wiring or schematic of the PCB

### ISSUE THREAD

**Issue by cparrapa** [JackGruber/OttoDIY_Biped#1]
Hi @JackGruber 

First great work very interesting DIY PCB, do you have the wiring? 
4AA battery holder with a booster?, so how many batteries do you use? What input voltage? 
2 batteries in parallel? Not sure if the red booster is able to buck down from 6volt

**Comment by JackGruber** [comment]
Hi @cparrapa,

because i did not write my own software until now, i did not write the rebro completely. 
I have used 4 AA rechargeable batteries which have each about 1.2 V. 
This means that the supply voltage is approx. 4.8. The boost converter was to boots the VCC to 5V even if the batteries run out.

Sorry but i have no circuit diagram :(

**Comment by cparrapa** [comment]
No worries are you able to run 6 servos with that without resets or browns?

**Comment by JackGruber** [comment]
Yes with the cap it is possible, but with out boostconverter and alkaline battery (4* 1,5V) it works more stable.

**Comment by cparrapa** [comment]
Hi what cap is that 100uF? and what happens with 1.5V if in total is 6V and the booster supposed to deliver 5V?

**Comment by JackGruber** [comment]
The Elko is a 10V 1000uF. 
> what happens with 1.5V if in total is 6V and the booster supposed to deliver 5V?

It slightly reduces the voltage. At 6V I get about 5.8V

**Comment by cparrapa** [comment]
thanks i will give a try!
